client/client_test.go

```diff
@@ -1083,12 +1082,14 @@ func testFrontendImageNaming(t *testing.T, sb integration.Sandbox) {
 
 					switch exp {
 					case ExporterOCI:
+						integration.CheckFeatureCompat(t, sb, integration.FeatureOCIExporter)
 						t.Skip("oci exporter does not support named images")
 					case ExporterDocker:
 						outW, err := os.Create(out)
```

Comment:
We need to test this one in moby again. I don't think the `ExporterDocker` case is correct for example.

Reply:
Hum yes indeed, let's see: https://github.com/moby/moby/actions/runs/4418305803